packages/torch.Dockerfile

```diff
@@ -31,7 +31,7 @@ ENV PYTORCH_BUILD_NUMBER=1
 # For PyTorch, we need specifically mkl.
 ENV LIBRARY_PATH="$LIBRARY_PATH:/opt/conda/lib"
 ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/conda/lib"
-ENV TORCH_CUDA_ARCH_LIST="3.7;6.0;7.0+PTX;7.5+PTX"
```

Comment:
Removing the `3.7` CUDA architecture (used by K80 which we don't have anymore).

Failling build at:
```
#13 328.0 nvcc fatal   : Unsupported gpu architecture 'compute_37'
```